.github/workflows/python-integration-test.yml

```diff
@@ -66,7 +66,7 @@ jobs:
           for pythonVersion in "3.8" "3.9" "3.10" "3.11"
           do
             echo "Use Python version ${pythonVersion} to test the Python client."
-            ./gradlew -PjdkVersion=${{ matrix.java-version }} -PpythonVersion=${pythonVersion} :clients:client-python:test
+            ./gradlew -PjdkVersion=${{ matrix.java-version }} -PpythonVersion=${pythonVersion} -PskipDockerTests=false :clients:client-python:test
```

Comment:
`skipDockerTests` is currently in effect because of exclude Java classes with tag `@Tag("gravitino-docker-test")`, but there is no Java class in python, so proposing this change in a separate PR does not reflect the necessity of this parameter. It is recommended to change it in the PR where it actually takes effect.

Reply:
Got it, I'll add this change in the related PR.